common/decisions/wc_lordaeron_decisions.txt

```diff
@@ -0,0 +1,77 @@
+﻿
+develop_the_undercity = { 
+    ai_check_interval = 6
+	sort_order = 1
+	picture = "gfx/interface/illustrations/decisions/wc_undercity_throne_room_decision.dds"
+	desc = develop_the_undercity_desc
+	selection_tooltip = develop_the_undercity_tooltip
+	major = yes
+
+	is_shown = {
+		NOT = {
+			title:b_lordaeron = { has_variable = develop_the_undercity }
+		}
+		is_of_forsaken_like_culture = yes
+		has_title = title:c_lordaeron
+	}
+
+	is_valid = {
+		piety_level >= 2
+
+		prestige_level >= 2
+
+		religion = { is_in_family = rf_shadow }
+
+		is_of_forsaken_like_culture = yes
+	}
+
+	cost = { 
+		gold = 250
+		prestige = 500
+	}
+
+	effect = {
+		title:b_lordaeron = { 
+			set_variable = develop_the_undercity 
+			years = 7
```

Comment:
take another look at this, i made some changes that i think u'll like. just putting 7 years are screwing up the event because if you retake lordaeron, the completion event will still fire. so i added a boolean to it

Reply:
Check error.log. Either
1) remove `years = 7` and set variable forever
2) or make as i showed.
I am speaking about technical error

Reply:
saw this just now.